packages/ag-charts-enterprise/src/features/annotations/annotationProperties.ts

```diff
@@ -222,6 +223,14 @@ export function LineDash<T extends Constructor>(Parent: T) {
     return LineDashInternal;
 }
 
+export function LineStyle<T extends Constructor>(Parent: T) {
+    class LineStyleInternal extends Parent {
+        @Validate(LINE_STYLE, { optional: true })
+        lineStyle?: AnnotationLineStyleType;
+    }
+    return LineStyleInternal;
+}
+
```

Comment:
Should we merge the `LineDash()` mixin into this `LineStyle()` one? Can't see any situation where we'd have one and not the other, and they appear to always be set together?